deployment/live/gcp/cloudbuild/prod/terragrunt.hcl

```diff
@@ -0,0 +1,18 @@
+terraform {
+  source = "${get_repo_root()}/deployment/modules/gcp//cloudbuild"
+}
+
+locals {
+  docker_env = "ci"
+  server_docker_image = "${include.root.locals.location}-docker.pkg.dev/${include.root.locals.project_id}/docker-${local.docker_env}/conformance-gcp:latest"
```

Comment:
Is this required? I don't think it's being used.

Reply:
Removed.